### PR TITLE
Various improvements to cards

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
Starting with the easy bit:

image

Related: https://github.com/endlessm/kolibri-explore-plugin/issues/534

### CODE REVIEW

**Comment by jprvita** [comment]
I tested this and the play button was grey on the page listing the resource:
Screenshot from 2023-05-26 16-31-30
But clicking on the resource to go to the resource page (not sure if that is the right name) the play button is still enabled:
Screenshot from 2023-05-26 16-32-04

In addition, there are two things that seem wrong with the UX here:
1. If I try to click on the disabled button on the Library page, the click "falls through" and the effect is the same as if I had clicked on the card (navigating to the resource page), even if the cursor does not turn to the little hand (aka pointer) indicating that area is clickable;
2. In the resource page, the whole card is enabled -- perhaps this is just a side-effect of the button there being enabled too, but we should keep that in mind and address it at the same time.

**Comment by jofilizola** [comment]
> I tested this and the play button was grey on the page listing the resource: Screenshot from 2023-05-26 16-31-30 But clicking on the resource to go to the resource page (not sure if that is the right name) the play button is still enabled: Screenshot from 2023-05-26 16-32-04
Great catch!
In the future we want the resource page to be the same as upstream, so the behavior here would be:
Screenshot 2023-05-30 at 16 33 27

For now, two options:
Screenshot 2023-05-30 at 16 34 22

File: packages/ek-components/src/components/EkCardBody.vue [hunk 1]
```diff
@@ -1,27 +1,49 @@
...
-        {{ subtitle }}
-      </VClamp>
-    </p>
+    <v-container>
+
+      <!-- Channel -->
+      <v-row v-if="showChannelIcon">
+        <v-col>
+          <!-- FIXME only for debugging: -->
+          <span class="debug-node-id sr-only">{{ node.id }}</span>
```
Comment (manuq) [inline packages/ek-components/src/components/EkCardBody.vue:9]: I was about to suggest "remove this" but I see it was there before.. so I guess nevermind :)

File: packages/ek-components/src/components/EkContentCard.vue [hunk 2]
```diff
@@ -99,9 +106,18 @@ export default {
...
 .card {
   transition: all ease .4s;
-  &:hover {
+  &:hover:not(.disabled) {
     color: $primary;
   }
+  &.disabled {
+    background: transparent;
+    outline: solid 1px $gray-400;
+    outline-offset: -1px;
```
Comment (manuq) [inline packages/ek-components/src/components/EkContentCard.vue:115]: :+1:

File: packages/ek-components/src/components/EkPlayButton.vue [hunk 3]
```diff
@@ -105,4 +113,9 @@ export default {
   text-transform: uppercase;
 }
 
+button:disabled {
+  opacity: 0.33;
+  background-color: rgba(0, 0, 0, 0.25);
```
Comment (manuq) [inline packages/ek-components/src/components/EkPlayButton.vue:118]: Two options for de-hardcoding these values:
- Add new variables to the theme at `packages/ek-components/src/styles.scss` (if this is wanted only for EkPlayButton)
- Change the style of all disabled buttons in the theme at `packages/ek-components/src/index.scss` (if this is wanted for all disabled buttons)

File: packages/ek-components/src/styles.scss [hunk 4]
```diff
@@ -32,13 +32,13 @@ $modal-backdrop-opacity: 0.8;
...
-  "html5": $yellow,
-  "zim": $yellow,
-  "video": $orange
+  "bundle": $gray-300,
+  "audio": $gray-300,
+  "document": $gray-300,
+  "exercise": $gray-300,
+  "html5": $gray-300,
+  "zim": $gray-300,
+  "video": $gray-300
```
Comment (manuq) [inline packages/ek-components/src/styles.scss:41]: :+1: I guess it's not worth the effort refactoring this now as design decision to make all the same gray may change.

File: packages/ek-components/src/components/EkContentLink.vue [hunk 5]
```diff
@@ -17,14 +18,24 @@ export default {
...
   },
   emits: ['isHovered'],
   methods: {
     handleHover(hovered) {
       this.$emit('isHovered', hovered);
     },
     onClick() {
-      this.$router.push(this.url);
+      if (this.enabled)
+        this.$router.push(this.url);
```
Comment (manuq) [inline packages/ek-components/src/components/EkContentLink.vue:38]: :+1:

**Review by manuq — APPROVED** [review]
All looks good! Feel free to accept the de-hardcoding suggestion and merge.

File: packages/ek-components/src/components/EkPlayButton.vue (hunk 3 above)
Comment (GeorgesStavracas) [reply packages/ek-components/src/components/EkPlayButton.vue]: Done!

**Review by manuq — APPROVED** [review]

**Pull request merged**